mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -819,6 +819,24 @@ func ResourceContainerCluster() *schema.Resource {
 				Description: `Whether to enable Kubernetes Alpha features for this cluster. Note that when this option is enabled, the cluster cannot be upgraded and will be automatically deleted after 30 days.`,
 			},
 
+			"enable_k8s_beta_apis": {
```

Comment:
I think you'll need to wrap this in a `<% unless version == 'ga' -%>` / `<% end -%>` block to make it available to the "beta" version of the provider only. See `tpu_config` field below which is also beta-only.

Same applies for all of your other code blocks, including tests.

Reply:
That doesn't sound right, because the field is available in both versions of our API, any idea what could be wrong? I did  resync to main to pull in latest of main

Reply:
It could be the case that changes have not made it all the way to the provider. Were these changes made recently? I don't see the fields you're using at https://pkg.go.dev/google.golang.org/api/container/v1, which is the package used in our provider.